features/step_definitions/stream_steps.rb

```diff
@@ -14,6 +14,10 @@
   find(".toggle_nsfw_state", match: :first).click
 end
 
+When /^I toggle all nsfw posts$/ do
+  page.execute_script("$('a.toggle_nsfw_state').click()")
+end
```

Comment:
Unless you can indicate me how to click multiple values with Capybara. I did a quick search. I didn't found. the `find` method doesn't let me click multiple links. I tried `find(".toggle_nsfw_state").each{ |elt| elt.click }` I got `Capybara::Ambiguous`.